### PR TITLE
Removing PV usage percentage numbers from runbooks doc text

### PR DESCRIPTION
**Pull request by aruniiird**
As we are changing the alert percentages from newer ODF version onwards, making the runbooks documentation texts general, thus making runbooks not dependant on ODF versions.

### CODE REVIEW

**Comment by aruniiird** [comment]
@jmolmo , @weirdwiz , please take a look at the changes.
Thanks

File: alerts/openshift-container-storage-operator/PersistentVolumeUsageNearFull.md [hunk 1]
```diff
@@ -16,8 +17,9 @@ leading to data loss if not addressed promptly.
 ## Diagnosis
 
 The alert triggers when a Persistent Volume Claim (PVC) approaches or surpasses
-75% of its capacity. It indicates the need to expand the PVC size to accommodate
-more data or to remove unnecessary data to free up space.
+very high capacity limit. It indicates the need to expand the PVC size to
+accommodate more data or to remove unnecessary data to free up space.
+Please see the alert doc for exact threshold limit.
```
Comment (jmolmo) [inline alerts/openshift-container-storage-operator/PersistentVolumeUsageNearFull.md:22]: Why not use the same text explanation in all the runbooks?
_"Please see the alert documentation text for an exact threshold limit."_

File: alerts/openshift-container-storage-operator/PersistentVolumeUsageNearFull.md [hunk 2]
```diff
@@ -2,8 +2,9 @@
 
 ## Meaning
 
-Persistent Volume Claim (PVC) usage has exceeded 75% of its capacity,
+Persistent Volume Claim (PVC) usage is alarmingly very high,
 indicating an imminent risk of reaching full capacity.
+Please see the alert description for exact values.
```
Comment (jmolmo) [inline alerts/openshift-container-storage-operator/PersistentVolumeUsageNearFull.md:7]: Why not use the same text explanation in all the runbooks?
_"Please see the alert documentation text for an exact threshold limit."_

**Review by jmolmo — CHANGES_REQUESTED** [review]

**Review by jmolmo — APPROVED** [review]

File: alerts/openshift-container-storage-operator/PersistentVolumeUsageNearFull.md (hunk 1 above)
Comment (aruniiird) [reply alerts/openshift-container-storage-operator/PersistentVolumeUsageNearFull.md]: Done

File: alerts/openshift-container-storage-operator/PersistentVolumeUsageNearFull.md (hunk 2 above)
Comment (aruniiird) [reply alerts/openshift-container-storage-operator/PersistentVolumeUsageNearFull.md]: Done

**Comment by aruniiird** [comment]
@agarwal-mudit , it is already approved by @jmolmo , but it is not yet merged. Can you please take a look (and approve if any additional sanction is required)?

**Review by agarwal-mudit — APPROVED** [review]
/lgtm

**Pull request merged**